templates/types/streaming/nextjs/webpack.config.mjs

```diff
@@ -1,13 +1,15 @@
 // webpack config must be a function in NextJS that is used to patch the default webpack config provided by NextJS, see https://nextjs.org/docs/pages/api-reference/next-config-js/webpack
 export default function webpack(config) {
-  // See https://webpack.js.org/configuration/resolve/#resolvealias
-  config.resolve.alias = {
-    ...config.resolve.alias,
-    sharp$: false,
-    "onnxruntime-node$": false,
-  };
   config.resolve.fallback = {
     aws4: false,
   };
+
+  // Following lines will fix issues with onnxruntime-node when using pnpm
+  // See: https://github.com/vercel/next.js/issues/43433
+  config.externals.push({
```

Comment:
@thucpn do we need both (also `serverComponentsExternalPackages`)? what's the impact for npm?